OrbitLinuxTracing/GpuTracepointEventProcessor.cpp

```diff
@@ -4,6 +4,8 @@
 #include <tuple>
 #include <vector>
 
+#include "OrbitLinuxTracing/Events.h"
```

Comment:
```suggestion
#include <OrbitLinuxTracing/Events.h>
```

Reply:
About that - I think it is common practice to use <> only for C++ standard library and C system headers, for other libraries "" are used.

Reply:
Done.

Reply:
> About that - I think it is common practice to use <> only for C++ standard library and C system headers, for other libraries "" are used.

That's possible, my rule so far was to use <> for module-public headers because that's what libunwindstack was doing. But for example perfetto uses "".

Reply:
I don't have a strong opinion here. We can adapt this later?